src/EPR.PRN.Backend.Obligation.UnitTests/Services/RecyclingTargetDataServiceTests.cs

```diff
@@ -3,45 +3,46 @@
 using EPR.PRN.Backend.Obligation.Services;
 using FluentAssertions;
 using Moq;
+using System.Diagnostics.CodeAnalysis;
 
-namespace EPR.PRN.Backend.Obligation.UnitTests.Services
+namespace EPR.PRN.Backend.Obligation.UnitTests.Services;
+
+[ExcludeFromCodeCoverage]
```

Comment:
remove

Reply:
We don't need to look for code coverage on unit tests